static/templates/settings/account-settings.handlebars

```diff
@@ -132,5 +132,48 @@
             </div>
         </div>
 
+        <hr class="settings_separator" />
+
+        <div class="form-horizontal" id="api_key_button_box">
+            <div class="input-group side-padded-container">
+                <p class="api-key-note">
+                    {{#tr this}}
+                    For most bots using the API, you'll want to give each bot
+                    its own name and API key using the above section.  But if you
+                    want to write a bot that can access your own private
+                    messages, you should use your personal API key.
+                    {{/tr}}
```

Comment:
Does something like this make sense?

"Creating a bot, and using the bot's account / API key, is the recommended way to use the Zulip API.  However, you can also use your personal API key if you need to a way to automatically send and edit messages from your own account."